sqlglot/expressions.py

```diff
@@ -6130,6 +6100,10 @@ class CurrentUser(Func):
     arg_types = {"this": False}
 
 
+class LocalTime(Func):
```

Comment:
The `Func` subclass names have implicit semantics. If you do `LocalTime`, that means the parser will recognize and produce this expression for `LOCAL_TIME`, not `LOCALTIME`.

```suggestion
class Localtime(Func):
```